controller/iteration.go

```diff
@@ -37,7 +38,7 @@ func NewIterationController(service *goa.Service, db application.DB, config Iter
 
 // CreateChild runs the create-child action.
 func (c *IterationController) CreateChild(ctx *app.CreateChildIterationContext) error {
-	_, err := login.ContextIdentity(ctx)
+	currentUser, err := login.ContextIdentity(ctx)
```

Comment:
login.ContextIdentity(ctx) doesn't hit DB actually. It's just extracts the identity ID from the token from the context.